dev/gather_coverage.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# This file was modified from GAP-system's dev/ci-gather-coverage.sh.  Visit
+# their website at <https://www.gap-system.org/>.
```

Comment:
Thanks, I really appreciate that you want to give credit, but there is no need to do so for this trivial one liner :-). Not that I mind, either. Feel free to leave it or drop it (Abd just to make clear: the one-liner is trivial only in hindsight of course, I spent lots of time until I finally had it figured out *sigh*. I wish Codeoc included this kind of stuff in their docs)

Reply:
Haha, yes :-) I'm always uncertain about how much code one can copy...